tools/gn

```diff
@@ -328,12 +328,16 @@ def to_gn_args(args):
     # use it instead of building a new one.
     if args.prebuilt_dart_sdk:
       if can_use_prebuilt_dart(args):
+        print('Using prebuilt Dart SDK binary. If you are editing Dart sources '
+              'and wish to compile the Dart SDK, set `--no-prebuilt-dart-sdk`.')
         gn_args['flutter_prebuilt_dart_sdk'] = True
         gn_args['dart_sdk_output'] = 'built-dart-sdk'
       else:
-        print('--prebuilt-dart-sdk was specified, but an appropriate prebuilt '
-              'could not be found! Try running '
-              'flutter/tools/download_dart_sdk.py manually.')
+        print('Tried to download prebuilt Dart SDK but an appropriate version '
```

Comment:
This is only the right message if it's a build that would try to do a Dart SDK build. If you take a look in `can_use_prebuilt_dart()` at the end there's a check that the right prebuilt exists given the build configuration. I think we'd only want this warning if we expect a prebuilt to exist, but can't find it. Sorry this is complicated!

Reply:
No problem, sorry that reviewing this was probably harder than doing it yourself :)